generatedocs/data/params.json

```diff
@@ -463,7 +463,7 @@
         "Enable re-connection attempts if the WebSocket is disconnected. Defaults to false."
```

Comment:
Suggestion (to make the docs consistent):
"Enable re-connection attempts if the WebSocket is disconnected. Defaults to `false`."